tools/eubfr-cli/lib/extractCredentials.js

```diff
@@ -0,0 +1,11 @@
+const extractCredentials = ({ producerKey, credentials }) => {
+  let creds = '';
+  credentials.forEach(producerCredentials => {
```

Comment:
what about using `Array.find` instead?

Reply:
There are several methods to solve this, the question is what is the benefit to change to Array.find when it's readable and it works?

Reply:
```